djangoproject/templates/fundraising/includes/_hero_with_logo.html

```diff
@@ -2,10 +2,8 @@
 <div class="hero">
   <div class="hero-logo">
     {% if obj.url %}<a href="{{ obj.url }}" rel="nofollow">{% endif %}
-    <img src="{% static 'img/fundraising-heart.svg' %}"
-         data-src="{% if obj.logo %}{{ obj.thumbnail.url }}{% else %}{% static 'img/fundraising-heart.svg' %}{% endif %}"
-         data-src-retina="{% if obj.logo %}{{ obj.thumbnail.url|resolution:'2x' }}{% else %}{% static 'img/fundraising-heart.svg' %}{% endif %}"
-         alt="{{ obj.display_name }}" />
+    <img src="{% if obj.logo %}{{ obj.thumbnail.url }}{% else %}{% static 'img/fundraising-heart.svg' %}{% endif %}"
+         loading="lazy" alt="{{ obj.display_name }}" />
```

Comment:
I realize that the code was already there before you made your change, but since we're changing it I figured we might as well improve the alt text a little bit here, maybe something like:
```suggestion
    {% if obj.logo %}
        <img
            src="{{ obj.thumbnail.url }}"
            loading="lazy"
            alt="{% blocktranslate trimmed with company_name=obj.display_name %}
                Logo of company {{ company_name }}{% endblocktranslate %}"
        >
    {% else %}
        <img src="{% static 'img/fundraising-heart.svg' %}" alt="{% translate 'Pixelated heart logo' %}">
    {% endif %}
```

Reply:
Done!